spread-tests/main/mount-ns-layout/process.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import sys
+import json
+import re
+
+_boring_fs = set(['cgroup', 'fusectl', 'debugfs', 'pstore', 'securityfs', 'mqueue', 'hugetlbfs'])
```

Comment:
Ha, because I wanted to make it shorter (because there was an instability that I joust found and fixed). I'll amend this test to ignore nothing.